### PR TITLE
Issue #982 - Copied /examples folder from api-bench

### LINKED ISSUE #982: Move /examples to the examples repository

### PR DESCRIPTION
**Pull request by SlowMoschen**
this commit moved the /exmaples folder from api-bench repo into its own example repo

Closes https://github.com/idrinth-api-bench/api-bench/issues/982

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+env:
```
Comment (Idrinth) [inline .eslintrc.yml:176]: this needs to be merged with the same name file in the api-bench root - otherwise it won't work :)

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: .eslintrc.yml (hunk 1 above)
Comment (SlowMoschen) [reply .eslintrc.yml:176]: so this means I need to merge the eslintrc.yml from the the root repo into this yml?

Sorry if that is a stupid question, I am new to this kind of workflow. 😅

The eslintrc is the setup for the Typescript warnings and error sytem am I right?

Comment (Idrinth) [reply .eslintrc.yml:176]: > so this means I need to merge the eslintrc.yml from the the root repo into this yml?

Yes

> Sorry if that is a stupid question, I am new to this kind of workflow. 😅

No worries :)

> The eslintrc is the setup for the Typescript warnings and error sytem am I right?

Not quite, eslint is something on top, that helps detect issues with javascript and typescript.

Comment (SlowMoschen) [reply .eslintrc.yml:176]: I've made another commit, merging the two .yml files

**Review by Idrinth — APPROVED** [review]
copy over .coderabbit.yaml as well please :)

**Comment by SlowMoschen** [comment]
copying of coderabbit.yaml done - should I make the changes suggested from coderabbit?

**Comment by Idrinth** [comment]
> copying of coderabbit.yaml done - should I make the changes suggested from coderabbit?

no need, it is wrong with those :)

File: reqres.in/src/routes/main/single-user-not-found.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = (apiRootUrl,) => ({
```
[the remark this answers is not in the document]
Comment (Idrinth) [reply reqres.in/src/routes/main/single-user-not-found.js:1]: @coderabbitai the trailing comma is a requirement for the whole organisation

**Pull request merged**